llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/Passes/Passes.td

```diff
@@ -59,6 +59,17 @@ def TopkSplitReduction:
   ];
 }
 
+def LinalgExtPackOpVectorization:
+    Pass<"iree-linalg-ext-pack-op-vectorization", "func::FuncOp"> {
```

Comment:
I call it LinalgExtPackOpVectorization because the vectorization of unpack op is not clear to me. We could merge or rename the passes once we have clearer picture.

Reply:
Havent looked at the PR yet (probably later tonight or tomorrow), but I'd assume unpack vectorization would go similar. Unpack is transpose + collapse shape + extract_slice. With all the outer tile dimensions being 1, there would be no collapse shape. So just a transpose, and that get vectorized as other transposes.

Reply:
Good point! Yes, that's plan for vectorizing unpack ops. SG!